Addressbook/src/addressbook.sh

```diff
@@ -1,2 +1,62 @@
 #!/bin/ash
```

Comment:
Please see https://github.com/denesd/shell-script-practice/pull/1/files#r264617896 and https://github.com/denesd/shell-script-practice/pull/1/files#r264667552